.env

```diff
@@ -1,3 +1,5 @@
 GRAPHQL_SERVER=https://prodready-dev-graphql.herokuapp.com/graphql
 NEXT_PUBLIC_STRIPE_KEY=pk_test_51JyvqZSFWb1hhs6iXu6o7geRq7PrRQYyLzg99iyoqTA8cwK9ccI0sWFmp9mnX1sPlRBewzT9kA3Kc8IwtanApwIf00pR6tm02Y
-STRIPE_SECRET_KEY=sk_test_51JyvqZSFWb1hhs6ihlHe68hqn9s5XejitpYVd7obvZSk1h4V7XC8co2kU2GWcJwUU5JmduUAriaBkH5vI3YL5LxM00avaAbiSp
\ No newline at end of file
+STRIPE_SECRET_KEY=sk_test_51JyvqZSFWb1hhs6ihlHe68hqn9s5XejitpYVd7obvZSk1h4V7XC8co2kU2GWcJwUU5JmduUAriaBkH5vI3YL5LxM00avaAbiSp
+SUCCESS_URL=https://prodready-dev-21hezpbth-souvikbasu.vercel.app/success
```

Comment:
SUCCESS_URL=/success

While using it: "https://" + window.location.hostname + SUCCESS_URL